lib/api/src/lib.rs

```diff
@@ -451,10 +451,10 @@ mod sys;
 #[cfg(feature = "sys")]
 pub use sys::*;
 
-#[cfg(feature = "js")]
+#[cfg(feature = "web")]
 mod js;
```

Comment:
the feature is web but the mod stay `js`?

Reply:
Good call. Let me fix that